fixtures/mock-oauth2-provider/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2019.
```

Comment:
Please fix the header - I believe the year should be `2020`.

Reply:
agreed